packages/sdk/src/error-handling.ts

```
@@ -0,0 +1,404 @@
+    ErrorHandler.logError(bilanError)
+
+    if (ErrorHandler.isDebugMode) {
+      throw bilanError
+    }
+
+    return fallback
+  }
+}
+
+/**
+ * Graceful degradation helpers
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)